event-input.c

```diff
@@ -2510,6 +2539,8 @@ evin_iomon_activity_cb(mce_io_mon_t *iomon, gpointer data, gsize bytes_read)
     if( !ev || bytes_read != sizeof (*ev) )
         goto EXIT;
 
+    evin_iomon_user_feedback(ev);
```

Comment:
A bit of u-op ("keypress" and "activity" type input sourcs are not expected to emit too many events), but 
as a compromise: I moved the calls so that they are made only after EV_KEY has already been established so that we avoid double checking irrelevant events.